test/test-resize-event.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var cluster = require('cluster');
+var master = require('../lib/master');
+var pickWorker = require('./helpers').pickWorker;
+var tap = require('tap');
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
+});
+
+tap.test('should get resize event on return to configured size', function(t) {
+  function assertClusterResizesToConfiguredSizeAfter(somethingHappens, t) {
```

Comment:
This argument order feels wrong. Also, `doSomething` would be a better name I think.